src/context.rs

```diff
@@ -259,6 +260,52 @@ fn get_ext_tx_set(tx_size: TxSize, is_inter: bool,
     }
 }
 
+static intra_mode_to_tx_type_context: [TxType; INTRA_MODES] = [
+    DCT_DCT,    // DC
+    ADST_DCT,   // V
+    DCT_ADST,   // H
+    DCT_DCT,    // D45
+    ADST_ADST,  // D135
+    ADST_DCT,   // D117
+    DCT_ADST,   // D153
+    DCT_ADST,   // D207
+    ADST_DCT,   // D63
+    ADST_ADST,  // SMOOTH
+    ADST_DCT,   // SMOOTH_V
+    DCT_ADST,   // SMOOTH_H
+    ADST_ADST,  // PAETH
+];
+
+/*
+// enable with CfL
+static uv2y: [PredictionMode; UV_INTRA_MODES] = [
+    DC_PRED,        // UV_DC_PRED
+    V_PRED,         // UV_V_PRED
+    H_PRED,         // UV_H_PRED
+    D45_PRED,       // UV_D45_PRED
+    D135_PRED,      // UV_D135_PRED
+    D117_PRED,      // UV_D117_PRED
+    D153_PRED,      // UV_D153_PRED
+    D207_PRED,      // UV_D207_PRED
+    D63_PRED,       // UV_D63_PRED
+    SMOOTH_PRED,    // UV_SMOOTH_PRED
+    SMOOTH_V_PRED,  // UV_SMOOTH_V_PRED
+    SMOOTH_H_PRED,  // UV_SMOOTH_H_PRED
+    PAETH_PRED,     // UV_PAETH_PRED
+    DC_PRED,        // CFL_PRED
+];
+*/
+
+pub fn y_intra_mode_to_tx_type_context(pred: PredictionMode) -> TxType {
+    intra_mode_to_tx_type_context[pred as usize]
+}
+
+pub fn uv_intra_mode_to_tx_type_context(pred: PredictionMode)-> TxType {
+    intra_mode_to_tx_type_context[pred as usize]
+    //flip me when CfL is implemented
```

Comment:
Mark it as TODO (anyone): such that is easily searchable later?

Reply:
I have the other location marked as TODO so if you flip that you'll discover this one too.